python/pyspark/mllib/util.py

```diff
@@ -273,24 +317,30 @@ def convertVectorColumnsFromML(dataset, *cols):
         return callMLlibFunc("convertVectorColumnsFromML", dataset, list(cols))
 
     @staticmethod
-    @since("2.0.0")
     def convertMatrixColumnsToML(dataset, *cols):
         """
         Converts matrix columns in an input DataFrame from the
         :py:class:`pyspark.mllib.linalg.Matrix` type to the new
         :py:class:`pyspark.ml.linalg.Matrix` type under the `spark.ml`
         package.
 
-        :param dataset:
-          input dataset
-        :param cols:
-          a list of matrix columns to be converted.
-          New matrix columns will be ignored. If unspecified, all old
-          matrix columns will be converted excepted nested ones.
-        :return:
-          the input dataset with old matrix columns converted to the
-          new matrix type
+        .. versionadded:: 2.0.0
 
+        dataset : :py:class:`pyspark.sql.DataFrame`
+            input dataset
+        cols : str
```

Comment:
Is it a str or list of str?

Reply:
That's a good question. Looking at [`numpy.ndarray.item`](https://numpy.org/devdocs/reference/generated/numpy.ndarray.item.html) we should make varargs explicit

    *cols : ...

As of type I was thinking `str` as we support variable number of `str` values, but looking at the code, a single `List[str]` would do as well.

Reply:
>  a single List[str] would do as well.

Sorry, I took another look at the implementation and it turns out I've misread the code ‒ passing a `List[str]` won't work here. I'll fix that in a second.